Doc/whatsnew/3.12.rst

```diff
@@ -475,6 +480,12 @@ Pending Removal in Python 3.14
 * ``__package__`` and ``__cached__`` will cease to be set or taken
   into consideration by the import system (:gh:`97879`).
 
+* Testing the truth value of an :class:`xml.etree.ElementTree.Element`,
+  discouraged by :mod:`xml.etree.ElementTree` documentation since Python 2.6
+  and by :exc:`FutureWarning` in the Python implementation since Python 2.7,
```

Comment:
I don't think anyone ever actually saw the FutureWarning because the Python implementation never got used, lets skip mentioning that unseed oddity and just focus on it now triggering a DeprecationWarning.

Reply:
Smart. I _think_ the Python implementation was used before the C implementation became the default in Python 3.3, but I agree we can avoid the history deep dive.